src/scripts/main.js

```diff
@@ -2,4 +2,27 @@
 
 document.addEventListener('click', (e) => {
   // write code here
+  const spider = document.querySelector('.spider');
```

Comment:
It's not efficient to select the spider every time a click event occurs. It would be better to select the spider once and store it in a variable outside of the event listener.